node/operations.js

```diff
@@ -46,8 +46,8 @@ function Operations(opts) {
         out: []
     };
     self.requests = {
-        in: {},
-        out: {}
+        in: Object.create(null),
```

Comment:
we should perf setting and getting properies on `Object.create(null)`;

creating ops is O(n) connections; accessing ops is O(n) requests.